conan/tools/env/environment.py

```diff
@@ -170,12 +174,20 @@ def __repr__(self):
         return repr(self._values)
 
     def dumps(self):
-        """ returns a string with a profile-like original definition, not the full environment
-        values
+        """
+        :return: A string with a profile-like original definition, not the full environment
+                 values
         """
         return "\n".join([v.dumps() for v in reversed(self._values.values())])
 
     def define(self, name, value, separator=" "):
+        """
+        Overwrite previously existing `name` environment variable with new `value`
```

Comment:
I think I would not use overwrite here, just define... like this?
```suggestion
        Define `name` environment variable with value `value`
```

Reply:
Define ``name`` environment variable ...... If the variable ``name`` already existed, this definition will overwrite the existing value.

Or something like that.